engine/src/EntityFactory3D.cpp

```
@@ -174,6 +180,9 @@ namespace nexo
 
         components::StaticMeshComponent mesh;
         mesh.vao = renderer::NxRenderer3D::getTetrahedronVAO();
+        mesh.hasBounds = true;
```

Cppcheck (2.19.0)

[style] The function 'removeEventDebugFlags' is never used.

(unusedFunction)